packages/server-admin-ui/src/views/ServerConfig/BasicProvider.js

```diff
@@ -363,6 +363,39 @@ class BaudRateIntputCanboat extends Component {
   }
 }
 
+class StdOutInput extends Component {
+  constructor(props) {
+    super();
+    let value = props.value.toStdout;
+    if(Array.isArray(value)) {
+      value = value.join(',');
+    }
+    this.state = {value}
+    this.onChange = this.onChange.bind(this);
+  }
+  onChange(e) {
+    this.setState({value: e.target.value});
+    this.props.onChange({
+      target: {
+        type: e.target.type,
+        name: e.target.name,
+        value: e.target.value.split(','),
+      }
+    });
+  }
+  render () {
+    return (
+      <TextInput
+        title='Output Events'
+        name='options.toStdout'
+        helpText='Output events to this connection. Example: nmea0183,nmea0183out'
```

Comment:
I wonder how clear this is going to be to the users? Would something like `Events that should be written as output to this connection. Example: nmea0183,nmea0183out` just that bit more understandable?

Reply:
Sure, let's use that